test-integration/scripts/09-verify-format.sh

```diff
@@ -9,10 +9,11 @@ source $(dirname $0)/01-init-env.sh
 #-------------------------------------------------------------------------------
 # Run FORMATTING
 #-------------------------------------------------------------------------------
-dotnet tool install -g dotnet-format
-dotnet format --check
+# Commented lines below due to error in the formatting tool - see https://github.com/dotnet/format/issues/1519
```

Comment:
I had to comment `dotnet format` integration test because of the following error:
```
Unhandled exception: System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load file or assembly 'Microsoft.CodeAnalysis.CSharp, Version=4.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. O sistema não pode encontrar o arquivo especificado.
```


On further investigation, I found those open issues:
https://github.com/dotnet/format/issues/1519
https://github.com/dotnet/format/issues/1521

Summing up, it seems that the format tool is under is broken for .net 6 at the moment

Should I leave those lines commented and create another issue for `dotnet format` once a fix is ready?

Reply:
Can you try dotnet format --verify-no-changes rather than --check ?

Reply:
If it not work yes you can create an issue for later

Reply:
Maybe add the package Microsoft.CodeAnalysis.CSharp.CodeStyle like here https://betterprogramming.pub/enforce-net-code-style-with-editorconfig-d2f0d79091ac?source=rss----d0b105d10f0a---4 can resolve the error ?

Reply:
Even with --verify-no-changes it is not working.
I just commented the code so the integration tests would pass.
I will add the package Microsoft.CodeAnalysis.CSharp.CodeStyle and see if it works

Reply:
Same error even when adding the package.
Dotnet format seems to be broken for .net 6.0.2 for web projects. For console projects it is working (see https://github.com/dotnet/format/issues/1519)
Hopefully they will provide a fix soon, as none of the fixes I tried worked.

Reply:
Ok, no problem for me to fix it later you can merge when it's ok for you and i release a new version this week-end